src/lib.rs

```diff
@@ -118,7 +118,7 @@ impl<D: HasDisplayHandle, W: HasWindowHandle> Surface<D, W> {
     /// ## Platform Dependent Behavior
     ///
     /// - On X11, the window must be visible.
-    /// - On macOS, Redox and Wayland, this function is unimplemented.
+    /// - On Apple platforms, Redox and Wayland, this function is unimplemented.
```

Comment:
I _feel_ the term "Apple platforms" isn't particularly clear, I've never seen or used a term like this.
In Winit we also say "MacOS and iOS".

I assume you are doing this because it covers tvOS, watchOS and visionOS as well and I honestly don't have a good alternative, just thought I would leave my 2¢ here.

Reply:
Indeed, I didn't want to write all the platform names. But I guess it makes more sense to do so.

Reply:
Or maybe I'll just write AppKit and UIKit